examples/undocumented/python/classifier_domainadaptationsvm.py

```diff
@@ -4,15 +4,16 @@
 from shogun import StringCharFeatures, BinaryLabels, DNA
 from shogun import WeightedDegreeStringKernel
 from shogun import MSG_DEBUG
+from shogun import machine
 try:
 	from shogun import DomainAdaptationSVM
```

Comment:
Refactor like the one below too?

Reply:
yup, good catch. will try to remove it from swig as well

Reply:
Feel free to separate it and do it later in another PR.

Reply:
already dropped it :") also makes sense to be in this PR since it's dependent on SVMLight